sanitizer/search_fallback.js

```diff
@@ -1,29 +0,0 @@
-var sanitizeAll = require('../sanitizer/sanitizeAll'),
```

Comment:
I included it by accident in #942. It should have been deleted at the point of PR 941 with placeholder integration.